tests/generation/test_utils.py

```diff
@@ -2298,13 +2301,14 @@ def _test_attention_implementation(self, attn_implementation):
         NOTE: despite the test logic being the same, different implementations actually need different decorators, hence
         this separate function.
         """
-        max_new_tokens = 30
+        max_new_tokens = 3
```

Comment:
I think's it's fine to put something smaller than 30.

It also reduces the time when running with `--flake-finder --flake-runs=`